src/Simulation/CsharpGeneration.Tests/Circuits/UnitTests.qs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Core {
+    @Attribute ()
```

Comment:
I thought the suggested style was to not have a space between the attribute name and the inputs?